README.md

```diff
@@ -52,8 +52,6 @@ User alice = jsonb.fromJson(aliceJson, User.class);
 ## Links
 
 - Official web site: https://eclipse-ee4j.github.io/jsonb-api
-- JSON-B user guide: https://javaee.github.io/jsonb-spec/users-guide.html
```

Comment:
Can we replace this link with a new link for the user guide hosted at JakartaEE? This page is very important to users since it contains all of the JSON-B doc -- I also refer to it frequently.

Reply:
would filing a bug and re-adding a link once it is known work?

Reply:
https://github.com/eclipse-ee4j/jsonb-api/issues/149

Reply:
I'll publish it on the web site when I get back from vacation next week.

Reply:
Done it. See https://github.com/eclipse-ee4j/jsonb-api/pull/150